packages/scandipwa/src/component/MyAccountOrderTotals/MyAccountOrderTotals.component.js

```diff
@@ -56,7 +56,9 @@ export class MyAccountOrderTotals extends PureComponent {
     }
 
     renderDiscount({ label, amount: { value } }, index) {
-        return this.renderPriceLine(__('Discount (%s)', label), value, null, {}, `discount-${index}`);
+        const discountLabel = (label && (label !== 'Discount')) ? __('Discount (%s)', label) : __('Discount');
```

Comment:
It is hardcode, label can be translated and it will not be 'Discount' string in this case.